glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
the Garden API (which cloud foundry uses) has separate Create and Run calls (and can run multiple processes in a container, analogously to 'runC exec' or 'docker exec'). We'd ideally like, therefore, to maintain the flexibility to do this in the spec (as Trevor mentioned, this is already existing function in runC). We can work around not having this (e.g. By using runC directly and not supporting arbitrary OCS implementations, which would be a shame, or by waiting until the Run call to start the container which'd change our current semantics a bit) but I'd like to understand the reasoning behind not supporting these use cases in a common container spec.